build-support/docker/Verify-Release-Ubuntu.dockerfile

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+FROM ubuntu:22.04
```

Comment:
should we use `ubuntu:latest` here to avoid needing to remember to update in the future, or does that introduce other concerns?

Reply:
thank you!  resolved in 206971db6874891b7124e55617cf021c69910523